tools/utils/utils.cpp

```diff
@@ -8,32 +8,43 @@
 
 namespace evmc
 {
-bytes from_hex(const std::string& hex)
+namespace
+{
+inline int from_hex_digit(char h)
```

Comment:
Can't you just copy all this from fizzy? Or is it a copy from fizzy?

Reply:
Fizzy has a copy from evmc, except ported to C++17. This is a new thing.